driver/src/main.rs

```diff
@@ -184,6 +185,9 @@ struct Options {
         parse(try_from_str)
     )]
     use_shadowed_orderbook: bool,
+
+    #[structopt(long, env = "ORDERBOOK_FILE", parse(from_os_str))]
+    orderbook_file: Option<PathBuf>,
```

Comment:
If later we would store more than one file we should probably just pass a "cache directory" here and let each components handle their file names internally. Maybe not needed now, I'm just starting to get worried about all these little one use case specific config values.

Reply:
I prefer to introduce that if we actually have another thing we want to persist over pod recreations and not now.